workshop/views.py

```diff
@@ -35,7 +35,7 @@ class ClubDetailView(generics.RetrieveUpdateAPIView):
     Update the description of a Club (Partial Update).
     """
     # pylint: disable=no-member
-    queryset = Club.objects.all()
+    queryset = Club.objects.select_related('joint_secy','secy').all()
```

Comment:
@king-11 Can we use 'joint_secy' here? Similarly for 'joint_gensec'. These are ManyToMany fields.

Reply:
for many to many we have to use prefect_related I will add that up too

Reply:
> @king-11 Can we use 'joint_secy' here? Similarly for 'joint_gensec'. These are ManyToMany fields.

Okay will add that up

Reply:
I mean 'select_related' can only be used for ForeignKey and OneToOneField. So, maybe we can't use 'joint_secy' here, and similarly for 'joint_gensec' below.